src/next/components/Hero/Hero.tsx

```diff
@@ -0,0 +1,333 @@
+const HeroSplit = ({
+  alignment = "left",
+  backgroundColor = "black",
+  title,
+  subtitle,
+  buttonLabel,
+  buttonUrl,
+  secondaryButtonLabel,
+  secondaryButtonUrl,
+  backgroundUrl,
+}: HeroSplitProps) => {
+  const bgColor = backgroundColor === "black" ? "bg-canvas-black" : "bg-white"
+  const textColor = backgroundColor === "black" ? "text-white" : "text-black"
+
+  return (
+    <>
+      <section
+        className={`hidden md:flex h-[32rem] ${
+          alignment === "left" ? "flex-row" : "flex-row-reverse"
+        }`}
+      >
+        <div className={`h-full w-1/2 ${bgColor}`}>
+          <div
+            className={`flex flex-row p-10 items-center h-full ${
+              alignment === "left"
+                ? "justify-start text-start"
+                : "justify-end text-end"
+            } ${textColor}`}
+          >
+            <div className="flex flex-col gap-6">
+              <h1 className="text-6xl font-medium">{title}</h1>
+              {subtitle && <p className="text-2xl leading-9">{subtitle}</p>}
+              {buttonLabel && buttonUrl && (
+                <div
+                  className={`flex flex-row gap-4 ${
+                    alignment === "left" ? "justify-start" : "justify-end"
+                  }`}
+                >
+                  <Button
+                    colorVariant="white"
+                    label={buttonLabel}
+                    href={buttonUrl}
+                  />
+                  {secondaryButtonLabel && secondaryButtonUrl && (
+                    <Button
+                      colorVariant="black"
+                      label={secondaryButtonLabel}
+                      href={secondaryButtonUrl}
+                    />
+                  )}
+                </div>
+              )}
+            </div>
+          </div>
+        </div>
+        <div className="h-full w-1/2">
+          <div
+            className="bg-cover bg-center bg-no-repeat h-full"
+            style={{
+              backgroundImage: `url('${backgroundUrl}')`,
+            }}
+          ></div>
+        </div>
+      </section>
+      <section className="block md:hidden">
+        <HeroFloating
+          backgroundColor={backgroundColor}
+          title={title}
+          subtitle={subtitle}
+          buttonLabel={buttonLabel}
+          buttonUrl={buttonUrl}
+          secondaryButtonLabel={secondaryButtonLabel}
+          secondaryButtonUrl={secondaryButtonUrl}
+          backgroundUrl={backgroundUrl}
+        />
+      </section>
+    </>
+  )
+}
+
+const HeroFloating = ({
+  alignment = "left",
+  backgroundColor = "black",
+  title,
+  subtitle,
+  buttonLabel,
+  buttonUrl,
+  secondaryButtonLabel,
+  secondaryButtonUrl,
+  backgroundUrl,
+}: Omit<HeroFloatingProps, "variant">) => {
+  const bgColor = backgroundColor === "black" ? "bg-canvas-black" : "bg-white"
+  const textColor = backgroundColor === "black" ? "text-white" : "text-black"
+
+  return (
+    <section className="h-[32rem]">
```

Comment:
why is 32rem hardcoded? should it be a % of the screen height?

Reply:
It's a little difficult for us to scale vertically right now because it would cause the content to potentially not have enough space for smaller screens. Having this fixed height can guarantee sufficient space to a certain extent, since width cannot be controlled.

Reply:
Alright, just that seeing 32rem seems like a "magic number"